brainstormforce/ultimate-addons-for-gutenberg
# improvement: Updated Block SVGs and Colors.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/MRBCVVrLM5n/4190b52c-f3b8-4c02-b013-733f2bf65a7c?view=7d3bd44b-cfb9-4e77-a81b-e065d12bd460&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/P8u88WLL

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Open an editor.
- Check the UAGB category and block svg.

### Checklist:
